source/includes/administration/_billing_providers.md

```diff
@@ -106,6 +106,126 @@ Provider | Attributes | &nbsp;
 ---- | ---- | -----------
 all | `creditCards`<br/>*string* | The type of credit card accepted. This is a comma separated field.
 
+<!-------------------- CREATE BILLING PROVIDER -------------------->
+
+#### Create a billing provider
+
+`POST /billing_providers`
+
+Create a new billing provider.
+
+```shell
+# Creates a new billing provider
+curl -X POST "https://cloudmc_endpoint/rest/billing_providers" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+> Request body example:
+
+```js
+{
+  "name": "Chase Credit Card (Dev)",
+	"organization": {
+			"id": "23910576-d29f-4c14-b663-31d728ff49a5"
+	},
+	"type": "CREDIT_CARD",
+  "providerType": "chaseCreditCard",
+	"configurationAttributes": {
+		"creditCards": "Visa, Mastercard, American Express, Discover"
+	} 
+}
+```
+> The above command return JSON structured like this:
+
+```js
+{
+  "data": {
+    "createdDate": "2021-05-17T19:59:33.387453Z",
+    "organization": {
+      "id": "23910576-d29f-4c14-b663-31d728ff49a5"
+    },
+    "name": "testing API",
+    "configurationAttributes": {
+      "creditCards": "Visa, Mastercard, American Express, Discover"
+    },
+    "defaultForType": false,
+    "id": "8db7cd13-b309-4fb9-b022-1632dbd165d0",
+    "updatedDate": "2021-05-17T19:59:33.387477Z",
+    "type": "CREDIT_CARD",
+    "providerType": "chaseCreditCard"
+  }
+}
+```
+
+Required | &nbsp;
+---------- | -----------
+`name`<br/>*string* | The name of the billing provider object.
+`organization.id`<br/>*UUID* | UUID of the organization to which belongs the billing provider. This can only be a organization which is root or reseller.
+`type`<br/>*string* | The type of billing provider. Possible values are: `CREDIT_CARD`.
+`providerType`<br/>*string* | The provider for the associated type.
```

Comment:
missing the defaultForType attribute in an optional section

Reply:
da4d7b6

Reply:
da4d7b6